app/styles/_layout.scss

```diff
@@ -0,0 +1,248 @@
+	}
+
+	> .toast-warning {
+		background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAGYSURBVEhL5ZSvTsNQFMbXZGICMYGYmJhAQIJAICYQPAACiSDB8AiICQQJT4CqQEwgJvYASAQCiZiYmJhAIBATCARJy+9rTsldd8sKu1M0+dLb057v6/lbq/2rK0mS/TRNj9cWNAKPYIJII7gIxCcQ51cvqID+GIEX8ASG4B1bK5gIZFeQfoJdEXOfgX4QAQg7kH2A65yQ87lyxb27sggkAzAuFhbbg1K2kgCkB1bVwyIR9m2L7PRPIhDUIXgGtyKw575yz3lTNs6X4JXnjV+LKM/m3MydnTbtOKIjtz6VhCBq4vSm3ncdrD2lk0VgUXSVKjVDJXJzijW1RQdsU7F77He8u68koNZTz8Oz5yGa6J3H3lZ0xYgXBK2QymlWWA+RWnYhskLBv2vmE+hBMCtbA7KX5drWyRT/2JsqZ2IvfB9Y4bWDNMFbJRFmC9E74SoS0CqulwjkC0+5bpcV1CZ8NMej4pjy0U+doDQsGyo1hzVJttIjhQ7GnBtRFN1UarUlH8F3xict+HY07rEzoUGPlWcjRFRr4/gChZgc3ZL2d8oAAAAASUVORK5CYII=") !important;
+	}
+}
+
+.toast-title {
+    font-weight: bold;
+}
+
+.toast-message {
+    -ms-word-wrap: break-word;
+    word-wrap: break-word;
+
+    a, label {
+        color: #ffffff;
+    }
+
+    a:hover {
+        color: #cccccc;
+        text-decoration: none;
+    }
+}
+
+.toast-close-button {
+    position: relative;
+    right: -0.3em;
+    top: -0.3em;
+    float: right;
+    font-size: 20px;
+    font-weight: bold;
+    color: #ffffff;
+    -webkit-text-shadow: 0 1px 0 #ffffff;
+    text-shadow: 0 1px 0 #ffffff;
+
+    &:hover,
+    &:focus {
+        color: #000000;
+        text-decoration: none;
+        cursor: pointer;
+    }
+}
+
+.toast {
+	position: relative;
+	background-color: #030303;
+
+	&::before {
+		content: '';
+		position: absolute;
+		background-color: rgba(255, 255, 255, .5);
+		height: 6px;
+		bottom: 0px;
+		left: 0px;
+		transition: width 9.8s linear;
+		width: 300px;
+	}
+
+	&.ng-enter::before {
+		transition: width 0s linear !important;
+		width: 0px !important;
+	}
+
+	&.ng-enter-active::before {
+		transition: width 0s linear !important;
+		width: 0px !important;
+	}
+
+	&.stop-timer::before {
+		transition: width 0s linear !important;
+		width: 0px !important;
+	}
+}
+
+.toast-top-full-width {
+    top: 0;
+    right: 0;
+    width: 100%;
+}
+
+.toast-bottom-full-width {
+    bottom: 0;
+    right: 0;
+    width: 100%;
+}
+
+.toast-top-left {
+    top: 12px;
+    left: 12px;
+}
+
+.toast-top-center {
+    top: 12px;
+}
+
+.toast-top-right {
+    top: 12px;
+    right: 12px;
+}
+
+.toast-bottom-right {
+    right: 12px;
+    bottom: 12px;
+}
+
+.toast-bottom-center {
+    bottom: 12px;
+}
+
+.toast-bottom-left {
+    bottom: 12px;
+    left: 12px;
+}
+
+.toast-center {
+    top: 45%;
+}
+
+.toast-success {
+	background-color: #51a351;
```

Comment:
That's the plan. A lot of this is half-baked, good suggestions though keep 'em coming.